spack/wdmapp/packages/wdmapp/package.py

```diff
@@ -28,10 +28,11 @@ class Wdmapp(BundlePackage):
             description='Build TAU version needed for performance analysis of the coupled codes')
 
     # FIXME this is a hack to avoid Spack not finding a feasible hdf5 on its own
-    # CWS - If the variant is not specified then there is a concretization error
-    # associated with hdf5.
-    # I think this is related to https://github.com/spack/spack/issues/15478 .
+    # CWS - If the variant is not specified for hdf5 and effis then there are
+    # concretization errors associated with hdf5 and python. I think this is related 
+    # to https://github.com/spack/spack/issues/15478 .
     depends_on('hdf5 +hl', when='~passthrough')
+    depends_on('effis', when='~passthrough')
```

Comment:
So is this also affected by the same spack bug, so that you have to list the `depends_on` conditionally twice, rather than just unconditionally once?

Reply:
That is my rough understanding.  Note, I only spent ~20 minutes poking at it and once it appeared to follow the same behavior that the `hdf5` spec was exhibiting... I deployed the kludge.  Do we have a 'direct' contact with someone spack team that could take a look at this if we can't find a cleaner fix?

Reply:
I can live with the workaround, I just wanted to make sure that it's actually needed and not just came via copy&paste. I don't have any direct contacts with spack (beyond their slack channel).